shared/getting-started/nextSteps/_default.md

```diff
@@ -0,0 +1,12 @@
+- Try out [local mode][local-mode], the most efficient way to rapidly develop and test your {{ $names.company.lower }} application.
+- Build an application that uses [multiple containers][multicontainer].
```

Comment:
I think we should focus more on the benefits of multicontainers here, as it might not be clear to everyone.